CONTRIBUTING.md

```diff
@@ -73,16 +75,16 @@ important part is that **a single commit maps to a single fix or enhancement**.
 Here are a few scenarios:
 
 * If a pull request adds a feature but also fixes two bugs, then the pull
-request should have three commits, one commit each for the feature and two bug
-fixes.
+  request should have three commits, one commit each for the feature and two bug
+  fixes.
 * If a PR is opened with 5 commits that was work involved to fix a single issue,
-it should be rebased to a single commit.
+  it should be rebased to a single commit.
 * If a PR is opened with 5 commits, with the first three to fix one issue and
-the second two to fix a separate issue, then it should be rebased to two
-commits, one for each issue.
+  the second two to fix a separate issue, then it should be rebased to two
+  commits, one for each issue.
 * Your pull request should be rebased against the current master branch. Please
-do not merge the current master branch in with your topic branch, nor use the
-Update Branch button provided by GitHub on the pull request page.
+  do not merge the current master branch in with your topic branch, nor use the
+  Update Branch button provided by GitHub on the pull request page.
```

Comment:
Update branch button is no longer there :), Nadia and I changed a Github Setting to prevent it from showing up.

Reply:
Closing out this comment to move to a maintenance ticket.